bindings.go

```diff
@@ -0,0 +1,193 @@
+package libovsdb
+
+import (
+	"fmt"
+	"reflect"
+)
+
+// nativeTypeFromExtended returns the native type that can hold a value of an
+// Extended type
+func nativeTypeFromExtended(extended ExtendedType) (reflect.Type, error) {
+	switch extended {
+	case TypeInteger:
+		return reflect.TypeOf(0), nil
```

Comment:
please create unexported var for all such types, so for each call nativeTypeFromExtended we don't need to call reflect again and again

Reply:
Good idea, will do